app/templates/teachers/teacher-student-view.pug

```diff
@@ -293,6 +293,9 @@ mixin breadcrumbs
                     span (
                     span(data-i18n="teacher.level_in_progress")
                     span )
+                  div
+                    span(data-i18n="play_level.changed")
+                    span= view.levelProgressTimeMap[level.get('original')].changed
```

Comment:
I wonder if we could get this on the same line, with a shorter date format, to tighten up the results. Maybe like relative moment time string, "edited 7 days ago" in small text on same line with the fully date as title attribute, and the date first completed also in the title?